config.py

```diff
@@ -23,6 +26,9 @@
 MAIL_PORT = 25
 MAIL_USERNAME = None
 MAIL_PASSWORD = None
+SENDER = 'no-reply@kimellab.ca'
```

Comment:
Should we move `MAIL_SERVER` and `SENDER` to config file?

Reply:
Yeah that might be a good idea. I dumped it here because at the moment none of our datman scripts send emails, but at some point that may change